### PR TITLE
Add SST dataset, corresponding dataloader and model.

### PR DESCRIPTION
**Pull request by heyLinsir**

### CODE REVIEW

File: cotk/dataloader/sentence_classification.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+class SST(SentenceClassification):
+	'''A dataloader for preprocessed SST dataset.
+
+	Arguments:
+			file_id (str): a str indicates the source of SST dataset.
+			file_type (str): a str indicates the type of SST dataset. Default: "SST"
+			valid_vocab_times (int): A cut-off threshold of valid tokens. All tokens appear
+					not less than `min_vocab_times` in **training set** will be marked as valid words.
+					Default: 10.
+			max_sen_length (int): All sentences longer than `max_sen_length` will be shortened
```
Comment (hzhwcmhf) [inline cotk/dataloader/sentence_classification.py]: max_sent_length

File: cotk/metric/metric.py [hunk 2]
```diff
@@ -43,6 +44,22 @@ class MetricBase(LoadClassInterface, metaclass=DocStringInheritor):
 		FORWARD_REFERENCE_ALLVOCABS_ARGUMENTS.replace("reference_allvocabs_key", \
 			"resp_allvocabs_key")
 
+	LABEL_KEY_ARGUMENTS = \
+		r"""label_key (str):
+			The key of reference sentence labels. Default: ``label``."""
+	LABEL_ARGUMENTS = r"""* **data[label_key]** (list or :class:`numpy.ndarray`):
+				  A 1-d jagged or padded array of int.
```
Comment (hzhwcmhf) [inline cotk/metric/metric.py]: A 1-d array of int

File: cotk/metric/metric.py [hunk 3]
```diff
@@ -43,6 +44,22 @@ class MetricBase(LoadClassInterface, metaclass=DocStringInheritor):
...
+				  Size: ``[batch_size]``,
+				  each element refers to label of one example"""
```
Comment (hzhwcmhf) [inline cotk/metric/metric.py]: example -> sample

File: models/SST-pytorch/run.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
+# coding:utf-8
+
+def run():
```
Comment (hzhwcmhf) [inline models/SST-pytorch/run.py]: refer to latest run
